lib/rspec/core/configuration.rb

```diff
@@ -318,6 +318,11 @@ def treat_symbols_as_metadata_keys_with_true_values=(_value)
       # Currently this will place a mutex around memoized values such as let blocks.
       add_setting :threadsafe
 
+      # @macro add_setting
+      # Maximum count of failed source lines to display in the failure reports.
+      # (default `10`).
+      add_setting :max_displayed_count_of_failed_lines
```

Comment:
This is subjective, but I think I slightly prefer `max_displayed_failure_line_count` as the name for this.  What do others think?